src/app/_constants/paths.ts

```diff
@@ -43,19 +44,23 @@ function createPathObject({
   includesEntries = false,
   customMainContentPath,
   customEntriesContentPath,
+  hasSubpaths = false,
 }: PathObjectParams): PathConfig {
-  const config: PathConfig = {
+  const mainContentPath = hasSubpaths
+    ? `${CONTENT_PAGES_ROOT}${path}${path}`
```

Comment:
I get what you want to do, but it took me a while to understand the line below.

`${CONTENT_PAGES_ROOT}${path}${path}`

If we want to _match the folder structure of Next.js app router_, how about we do this:

`${CONTENT_PAGES_ROOT}${path}/index.md`, where `index.md` is like the `page.tsx`.

If it requires big changes for this PR, nevermind for now.

Reply:
Yes, it would require to big of a change. Perhaps it would be too ambitious to match the folder structure of Next.js. More of an organizational choice to help group all security pages in this case.